docs/v1/use-cases.md

```diff
@@ -16,6 +16,18 @@ implement.
     2.  As the operator of an existing service running in Kubernetes, I want to
         be able to publish my services using a service broker, so that users
         external to my Kubernetes cluster can use my service
+        
+2. Searching and Browsing Services:
+    1.  As a consumer, I'm able to search my catalog for services by attributes
+        such as category.
+    2.  As a consumer, I'm able to see metadata about a service prior to 
+        creation which allows me to see if this service fits my need.
+    3.  As a consumer, I'm able to see all the required and optional parameters
+        I'll need to pass to the service in order to create it.
+    4.  As a consumer, when listing services I see a union of the catalogs
+        from all brokers I have registered. However, if I want to restrict the
+        list to a specific broker I can pass that in as a flag.
```

Comment:
I'm ok with call this out as a separate usecase, but it feels like a subset of the first one.